### DISCUSSION TITLE
Is it possible to add roop or similar tools in Fooocus?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#1483] [General]
Is it possible to add roop or similar tools in Fooocus?

**Comment by xhoxye** [comment]
faceswap? Already, in the input image, image prompt, advanced

**Reply by martinenkoEduard** [reply]
But it is not really good.
The face is not the same.

**Comment by lllyasviel** [comment]
Hi is there any special reason why you want to use that independent tool in Fooocus?
What is the difference between using that software inside or outside Fooocus?
